gems/pending/metadata/linux/OpenstackConfigurationParser.rb

```diff
@@ -0,0 +1,200 @@
+module MiqLinux
+  class OpenstackConfigurationParser
```

Comment:
@roliveri hm it's not part of the ems_refresh either. So I am guessing gems/pending/openstack/openstack_configuration_parser.rb ?